core/src/main/kotlin/com/alecstrong/sql/psi/core/mysql/MySql.bnf

```diff
@@ -80,7 +82,13 @@ approximate_numeric_data_type ::= 'FLOAT' | 'REAL' | ( 'DOUBLE' 'PRECISION' ) |
 
 date_data_type ::= 'DATE' | 'TIME' | 'DATETIME' | 'TIMESTAMP' | 'YEAR'
 
-string_data_type ::= 'CHAR' | 'VARCHAR' | 'TINYTEXT' | 'TEXT' | 'MEDIUMTEXT' | 'LONGTEXT' | 'ENUM' | 'SET'
+character_type ::= ( 'CHAR' | 'VARCHAR' | 'TINYTEXT' | 'TEXT' | 'MEDIUMTEXT' | 'LONGTEXT' )
+  [ '(' {signed_number} ')' ]
+  [ 'CHARACTER' 'SET' {identifier} ]
```

Comment:
I wanted this to be
```kotlin
  [ 'CHARACTER' 'SET' {charset_name} ]
```

But when I tried defining it in this file like
```
charset_name ::= id
```

It didn't work, so I fell back to `{identifier}`

Reply:
you might need to static import `id` above like you do with tokens

otherwise you could do `charset_name ::= {identifier}` which would probably work too